RecoBTag/TensorFlow/plugins/DeepDoubleXTFJetTagsProducer.cc

```diff
@@ -238,6 +265,7 @@ void DeepDoubleBTFJetTagsProducer::produce(edm::Event& iEvent, const edm::EventS
 
       // jet and other global features
       const auto & features = tag_infos->at(jet_n).features();
+      if (features.empty()) continue ;
```

Comment:
it appears that this has no effect on CPU, even though there is clearly fewer cases with filled values.

In the current configuration ```n_batch_jets = 1, n_batches = n_jets``` and this inner loop has a range of 1 iteration.
This means that ```tensorflow::run``` is still called for every jet, albeit with zeroed-out inputs.

Please correct
 (otherwise the effort to skip the jets has no utility).
Sorry, I should've noticed this already in the last tests over the weekend.

Reply:
Sorry about that, i've added a check to only run the batch if there are jet's in it that have features filled.